src/processor.rs

```diff
@@ -161,6 +161,7 @@ fn process_initialize_holder_rewards_pool(
 
         // Write the data.
         let mut data = extra_metas_info.try_borrow_mut_data()?;
+        msg!("JOE: I think it's failing here...");
```

Comment:
:eyes:

Reply:
Hahaha whoops! That was for the `lazy_static` issue.